test/unit/cff_parser_spec.js

```diff
@@ -368,6 +351,15 @@ describe('CFFParser', function() {
 });
 
 describe('CFFCompiler', function() {
+
+  function testParser(bytes) {
+    bytes = new Uint8Array(bytes);
+    return new CFFParser({
+      getBytes: () => {
+        return bytes;
+    }}, {}, SEAC_ANALYSIS_ENABLED);
```

Comment:
Nit: According to https://travis-ci.org/mozilla/pdf.js/builds/289770606#L532-L533 this fails linting, it probably needs to look something like this (which also reads a bit easier to me):
```javascript
  return new CFFParser({
    getBytes: () => {
      return bytes;
    },
  }, {}, SEAC_ANALYSIS_ENABLED);
```